modules/kimberliteBidAdapter.js

```diff
@@ -0,0 +1,72 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { ortbConverter } from '../libraries/ortbConverter/converter.js'
+
+const VERSION = '1.0.0';
+
+const BIDDER_CODE = 'kimberlite';
+const METHOD = 'POST';
+const ENDPOINT_URL = 'https://kimberlite.io/rtb/bid/pbjs';
+
+const converter = ortbConverter({
+  context: {
+    mediaType: BANNER,
+    netRevenue: true,
+    ttl: 300
+  },
+
+  request(buildRequest, imps, bidderRequest, context) {
+    const bidRequest = buildRequest(imps, bidderRequest, context);
+    bidRequest.site ||= {};
+    bidRequest.site.publisher ||= {
+      domain: bidderRequest.refererInfo.domain
```

Comment:
So we only want to set `site.publisher.domain` if the incoming `publisher` object was not defined yet?

If ORTB converter populates `publisher`, with something like

```js
{
   url: 'some url'
}
```

Then we would not send `domain` at all.

Reply:
That's a miss yes, thank you! We definitely should fix this.

Reply:
I would recommend just doing a deepSetValue of the values you want to overwrite / set.